data/migrations/V0059.1__public_audit_case_mv.sql

```diff
@@ -0,0 +1,752 @@
+SET search_path = public, pg_catalog;
+
+--
+-- 1)Name: ofec_audit_case_mv; Type: MATERIALIZED VIEW; Schema: public; Owner: fec
+--
+
+DROP MATERIALIZED VIEW ofec_audit_case_mv;
+CREATE MATERIALIZED VIEW ofec_audit_case_mv AS
+WITH 
+cmte_info AS (
+    SELECT DISTINCT dc.cmte_id,
+        dc.cmte_nm,
+        dc.fec_election_yr,
+        dc.cmte_dsgn,
+        dc.cmte_tp,
+        b.FILED_CMTE_TP_DESC::text AS cmte_desc
+    FROM auditsearch.audit_case aa, disclosure.cmte_valid_fec_yr dc, staging.ref_filed_cmte_tp b
+    WHERE btrim(aa.cmte_id) = dc.cmte_id 
+      AND aa.election_cycle = dc.fec_election_yr
+      AND dc.cmte_tp IN ('H', 'S', 'P', 'X', 'Y', 'Z', 'N', 'Q', 'I', 'O')
+      AND dc.cmte_tp = b.filed_cmte_tp_cd
+), 
+cand_info AS (
+    SELECT DISTINCT dc.cand_id,
+        dc.cand_name,
+        "substring"(dc.cand_name::text, 1,
+            CASE
+                WHEN strpos(dc.cand_name::text, ','::text) > 0 THEN strpos(dc.cand_name::text, ','::text) - 1
+                ELSE strpos(dc.cand_name::text, ','::text)
+            END) AS last_name,
+        "substring"(dc.cand_name::text, strpos(dc.cand_name::text, ','::text) + 1) AS first_name,
+        dc.fec_election_yr
+    FROM auditsearch.audit_case aa JOIN disclosure.cand_valid_fec_yr dc
+    ON (btrim(aa.cand_id) = dc.cand_id AND aa.election_cycle = dc.fec_election_yr)
+),
+audit_case_finding_info AS (
+    SELECT
+        audit_case_finding.audit_case_id::integer AS audit_case_id,
+        audit_case_finding.parent_finding_pk::integer AS primary_category_id,
+        audit_case_finding.child_finding_pk::integer AS sub_category_id
+    FROM auditsearch.audit_case_finding
+),
+audit_case_init AS (
+    SELECT
+        '-1'::integer AS primary_category_id,
+        '-2'::integer AS sub_category_id,
+        ac.audit_case_id::integer AS audit_case_id,
+        ac.election_cycle::integer AS cycle,
+        ac.cmte_id AS committee_id,
+        cmte_info.cmte_nm AS committee_name,
+        cmte_info.cmte_dsgn AS committee_designation,
+        cmte_info.cmte_tp AS committee_type,
+        cmte_info.cmte_desc AS committee_description,
+        ac.far_release_date::date AS far_release_date,
+        ac.link_to_report,
+        ac.audit_id::integer AS audit_id,
+        ac.cand_id AS candidate_id,
+        coalesce(cand_info.cand_name, 'N/A'::text) AS candidate_name
+    FROM auditsearch.audit_case ac
+    LEFT JOIN cmte_info ON cmte_info.cmte_id::text = ac.cmte_id::text 
+    AND cmte_info.fec_election_yr = ac.election_cycle
+    LEFT JOIN cand_info ON cand_info.cand_id::text = ac.cand_id::text 
+    AND cand_info.fec_election_yr = ac.election_cycle
+),
+audit_case_sebset AS (
```

Comment:
Is this audit_case_sebset the correct column name you want? (audit_case_subset)

Reply:
audit_case_sebset is subquery name, not column name.